doc/classes/Image.xml

```diff
@@ -612,6 +614,7 @@
 			OpenGL texture format [code]RGBA[/code] with four components, each with a bitdepth of 4.
 		</constant>
 		<constant name="FORMAT_RGB565" value="7" enum="Format">
+			OpenGL texture format [code]RGB[/code] with three components. Red and blue have a bitdepth of 4, and green has a bitdepth of 5.
```

Comment:
```suggestion
			OpenGL texture format [code]RGB[/code] with three components. Red and blue have a bitdepth of 5, and green has a bitdepth of 6.
```

Reply:
I don't know why I wrote it by 1 off honestly.